crates/ruff_linter/src/rules/ruff/rules/unused_variable_accessed.rs

```diff
@@ -0,0 +1,124 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_python_semantic::Binding;
+use ruff_python_stdlib::builtins::is_python_builtin;
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for usages of variables marked as unused (variable names starting with an underscore, except '_') in functions.
+/// Forbid using method and function variables that are marked as unused.
```

Comment:
I'm not sure this sentence in the docs adds much, as you explain the rule pretty well in the next paragraph. I'd take it out:

```suggestion
```